jinahub/indexers/searcher/FaissSearcher/faiss_searcher.py

```diff
@@ -62,6 +63,7 @@ def __init__(
         normalize: bool = False,
         nprobe: int = 1,
         dump_path: Optional[str] = None,
+        prefech_size: Optional[int] = None,
```

Comment:
This should be a non-zero int by default, right? `None` doesn't help

Reply:
here, `None` means load the full data at once. Yes, `None` will disable prefetch.